src/utils/config.js

```diff
@@ -14,6 +14,8 @@ export function getConfig() {
         gooddollar_contract: 'gooddollar-v1.i-am-human.near',
         new_sbt_contract: 'sbt1.i-am-human.testnet',
         api_link: 'https://api-ophc7vkxsq-uc.a.run.app',
+        fractal_link: 'https://app.fractal.id',
+        fractal_client_id: 'D6SgXZQdWYk0D8ILkIGpNK75ufFpxD0Mp9sHFb_2oM8',
```

Comment:
should this be visible to everyone? or should we use ENV?

Reply:
> @Megha-Dev-19 

please push this MR, I will take into account all your suggestions and in the next MR I will make changes

Reply:
> should this be visible to everyone? or should we use ENV?

we can use the key here, because it`s public key